<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client_vip</class>
 <widget class="QWidget" name="client_vip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>426</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>华康方圆体W7</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>会员办理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sdufe.qrc">
    <normaloff>:/山财大校徽.png</normaloff>:/山财大校徽.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>退出办理界面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>确认办理</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入您的用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华康方圆体W7</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入您的手机号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华康方圆体W7</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>手机号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华康方圆体W7</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>成为会员折扣多多，礼物多多</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="sdufe.qrc"/>
 </resources>
 <connections/>
</ui>
